apiserver/params/network.go

```diff
@@ -462,14 +467,19 @@ type EntitiesPortRanges struct {
 // plus on-machine detection.
 // There are cases when we convert it *back* to the ip/mask form anyway.
 type Address struct {
-	Value           string `json:"value"`
-	CIDR            string `json:"cidr,omitempty"`
-	Type            string `json:"type"`
-	Scope           string `json:"scope"`
-	SpaceName       string `json:"space-name,omitempty"`
-	ProviderSpaceID string `json:"space-id,omitempty"`
-	ConfigType      string `json:"config-type,omitempty"`
-	IsSecondary     bool   `json:"is-secondary,omitempty"`
+	Value             string `json:"value"`
+	CIDR              string `json:"cidr,omitempty"`
+	Type              string `json:"type"`
+	Scope             string `json:"scope"`
+	SpaceName         string `json:"space-name,omitempty"`
+	ProviderAddressID string `json:"provider-address-id,omitempty"`
```

Comment:
Let's drop these additions from this patch, and just land the changes to core/network.